modules/functional-test/testng_integrationTests.xml

```diff
@@ -97,13 +97,13 @@
             <class name="com.intuit.wasabi.tests.service.IntegrationPages"/>
         </classes>
     </test>
-    
+    <!--  
```

Comment:
The user permissions tests is failing and sudheer will look at it.  I've commented it from both wasabi and wasabi-intuit code.